Makefile

```diff
@@ -291,7 +291,7 @@ build/asm/%.o: asm/%.s
 build/assets/%.o: assets/%.c
 	$(CC) -c $(CFLAGS) $(MIPS_VERSION) $(OPTFLAGS) -o $@ $<
 	$(OBJCOPY) -O binary $@ $@.bin
-	$(RM_MDEBUG)
+	@$(RM_MDEBUG)
```

Comment:
I think this should stay in the output: unlike the objcopy and odjdump, it directly affects something that goes into the compiled ELF.